Paymetheus/ViewModels/StakeMiningViewModel.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Paymetheus.Decred.Wallet;
+using Paymetheus.Framework;
+using Grpc.Core;
+using Paymetheus.Decred.Util;
+using System;
+using System.Windows;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace Paymetheus.ViewModels
+{
+    public class StakeMiningViewModel : ViewModelBase
+    {
+        public StakeMiningViewModel() : base()
+        {
+            FetchStakeInfoCommand = new DelegateCommand(FetchStakeInfoAction);
+            FetchStakeInfoCommand.Execute(null);
+        }
+
+        private StakeInfoProperties _stakeInfoProperties;
+        public StakeInfoProperties StakeInfoProperties
+        {
+            get { return _stakeInfoProperties; }
+            internal set { _stakeInfoProperties = value; RaisePropertyChanged(); }
+        }
+
+        private StakeDifficultyProperties _stakeDifficultyProperties;
+        public StakeDifficultyProperties StakeDifficultyProperties
+        {
+            get { return _stakeDifficultyProperties; }
+            internal set { _stakeDifficultyProperties = value; RaisePropertyChanged(); }
+        }
+
+        public ICommand FetchStakeInfoCommand { get; }
+
+        private async void FetchStakeInfoAction()
+        {
+            try
+            {
+                StakeInfoProperties = await App.Current.Synchronizer.WalletRpcClient.StakeInfoAsync();
+                StakeDifficultyProperties = await App.Current.Synchronizer.WalletRpcClient.StakeDifficultyAsync();
+            }
+            catch (RpcException ex) when (ex.Status.StatusCode == StatusCode.FailedPrecondition)
```

Comment:
This exception handler is missing all error handling and will just catch any errors without responding to them in some way.  In rare cases this is ok but there is not even a comment here about why it is being done.  I would err on removing this catch block.